warehouse/malware/tasks.py

```diff
@@ -29,6 +32,23 @@ def run_check(task, request, check_name, obj_id):
         raise task.retry(exc=exc)
 
 
+@task(bind=True, ignore_result=True, acks_late=True)
+def backfill(task, request, check_name, sample_rate=1):
```

Comment:
A docstring for this method would be nice (particularly mentioning that `sample_rate` is a ratio in `[0, 1]` and not in `[0, 100]`, which is what I thought before I noticed the `sample * 0.01` in the request handler 😅